### PR TITLE
Migrate Aruba to holiday groups

### PR DESCRIPTION
**Pull request by PPsyrius**
## Proposed change

- Migrate Aruba holidays to `holiday_groups` standard.
- Add l10n supports for `en_US`, `nl`, `pap`, `uk` localization; `uk`'s l10n need some double checking from @KJhellico .
 (Blocked by #1313)

## Type of change

- [ ] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [x] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/locale/en_US/LC_MESSAGES/AW.po [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+#. New Year's Day
+#: ./holidays/countries/aruba.py:53
+msgid "Aña Nobo"
+msgstr "New Year's Day"
+
+#. Betico Croes Day
+#: ./holidays/countries/aruba.py:61
+msgid "Dia di Betico"
+msgstr "Betico Croes Day"
```
Comment (KJhellico) [inline holidays/locale/en_US/LC_MESSAGES/AW.po:27]:
```suggestion
msgstr "Betico Day"
```
This is closer to the original and is also used in official sources.

File: holidays/locale/uk/LC_MESSAGES/AW.po [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+#. New Year's Day
+#: ./holidays/countries/aruba.py:53
+msgid "Aña Nobo"
+msgstr "Новий рік"
+
+#. Betico Croes Day
+#: ./holidays/countries/aruba.py:61
+msgid "Dia di Betico"
+msgstr "День Бетіко Крус"
```
Comment (KJhellico) [inline holidays/locale/uk/LC_MESSAGES/AW.po:27]:
```suggestion
msgstr "День Бетіко"
```

**Review by KJhellico — COMMENTED** [review]
It's just awesome! Especially the Ukrainian translation ❤️ 🇺🇦 Please look at little suggestions below.
For a complete "Netherlands set" we also need to migrate Curacao holidays…

File: holidays/locale/en_US/LC_MESSAGES/AW.po [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+#. Christmas Day
+#: ./holidays/countries/aruba.py:144
+msgid "Pasco di Nacemento"
+msgstr "Christmas Day"
+
+#. Boxing Day
+#: ./holidays/countries/aruba.py:150
+msgid "Di dos dia di Pasco di Nacemento"
+msgstr "Boxing Day"
```
Comment (KJhellico) [inline holidays/locale/en_US/LC_MESSAGES/AW.po:87]:
```suggestion
msgstr "Second Day of Christmas"
```
It's also closed to the original (as in the Netherlands). "Boxing Day" is typical for US/UK and related cultures.

**Comment by PPsyrius** [comment]
> It's just awesome! Especially the Ukrainian translation ❤️ 🇺🇦 Please look at little suggestions below. For a complete "Netherlands set" we also need to migrate Curacao holidays…

I'm working on that right now - we might also want to add support for `ISO 3166-1` for both of them later (`NL-AW` for Aruba and `NL-CW` for Curacoa)

**Comment by PPsyrius** [comment]
Yet another coverall error, but it should otherwise be fine now.

**Comment by PPsyrius** [comment]
Gonna switch Papiamento/Papiamentu's language code to `pp` as per outlined in https://www.creolica.net/The-Multiple-Facets-of-Papiamentu rather than marking it as a Portuguese dialect

**Comment by PPsyrius** [comment]
Seems all good now :tada:

**Review by arkid15r — COMMENTED** [review]
🇦🇼 one happy island

LGTM

File: holidays/locale/uk/LC_MESSAGES/AW.po [hunk 4]
```diff
@@ -0,0 +1,87 @@
+# Aruba holidays uk localization.
+# Authors: ~Jhellico <jhellico@gmail.com>, (c) 2023.
```
[the remark this answers is not in the document]
Comment (PPsyrius) [reply holidays/locale/uk/LC_MESSAGES/AW.po:2]: I mean @KJhellico was more or less the one who double-checked Ukranian locs. so a yes for me.

**Comment by arkid15r** [comment]
Thank you 👍

**Pull request merged**